### PR TITLE
Tooltip

### PR DESCRIPTION
**Pull request by sgaragagghu**
Known bug, tooltip doesn't follow the window.

### CODE REVIEW

**Comment by xrdavies** [comment]
@sgaragagghu Thank you for your great efforts!

**Comment by jonano614** [comment]
I know, this PR is already merged, but I will add some comments.

File: win/xdagwallet/ToolTip.cpp [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+    	m_rectText.right  = long_RectWidth;
+    	m_rectText.top    = t_Rect.Height() * 0.10; 
+   	m_rectText.bottom = long_RectHeight;
+#pragma warning( pop ) 
+            	
+	m_rgnRoundRect.CreateRectRgn(t_Rect.left+30, t_Rect.top, t_Rect.right-30, t_Rect.bottom);
+
+	CRgn rgnComb;
+	rgnComb.CreateRectRgn(t_Rect.left+30, t_Rect.top, t_Rect.right-30, t_Rect.bottom);
+	SetWindowRgn(rgnComb.operator HRGN(), TRUE);
```
Comment (jonano614) [inline win/xdagwallet/ToolTip.cpp]: Why do you need ".operator HRGN()"?

File: win/xdagwallet/xdagwalletDlg.h [hunk 2]
```diff
@@ -42,13 +42,13 @@ class CXDagWalletDlg : public CDialog {
...
 	CString _accountAddress;
-	CEditWalletAddr _accountAddressEdit;
+	CEditWalletAddrMine _accountAddressEdit;
 	CString _transferAmount;
 	CString _transferAddress;
 	CString _state;
 	CStatic _stateControl;
 	CEdit _transferAmountEdit;
-	CEdit _transferAddressEdit;
+	CEditWalletAddr _transferAddressEdit;
```
Comment (jonano614) [inline win/xdagwallet/xdagwalletDlg.h]: What is the purpose of CEditWalletAddr for transfer address?

File: win/xdagwallet/CEditWalletAddrMine.cpp [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+	HDC hdc = ::GetDC(this->m_hWnd);
+
+	TEXTMETRIC tm;
+	GetTextMetrics(hdc, &tm);
+
+	int nWidth = tm.tmAveCharWidth;
+	int nHeight = tm.tmHeight;
+
+	GetClientRect(&rect);
+	ToolTip = CToolTip::Show(pt, &rect, nWidth, nHeight,"Copied!", 1);
```
Comment (jonano614) [inline win/xdagwallet/CEditWalletAddrMine.cpp]: Each time somebody do double click CToolTip::Show creates a new instance of CToolTip class. How do you free memory?

**Pull request merged**